tests/tpch/test_correctness.py

```diff
@@ -0,0 +1,76 @@
+import pathlib
+
+import pandas as pd
+import pytest
+from distributed import LocalCluster
+
+from .generate_answers import generate as generate_answers
+from .generate_data import generate as generate_data
+
+VERIFICATION_SCALE = 1
+
+pytestmark = pytest.mark.tpch_correctness
+
+
+@pytest.fixture(scope="module")
+def answers_path(tmp_path_factory):
+    path = tmp_path_factory.mktemp("answers")
+    return generate_answers(base_path=path)
+
+
+@pytest.fixture(scope="module")
+def data_path(tmp_path_factory):
+    path = tmp_path_factory.mktemp("data")
+    scale = VERIFICATION_SCALE
+    return pathlib.Path(generate_data(scale=scale, path=str(path), relaxed_schema=True))
```

Comment:
Currently, we generate answers and data on the fly. We can adjust this if it strains the CI job, but right now it only adds ~15s overhead locally.